pandas/core/generic.py

```diff
@@ -684,6 +684,8 @@ def transpose(self, *args, **kwargs):
         new_axes = self._construct_axes_dict_from(self, [self._get_axis(x)
                                                          for x in axes_names])
         new_values = self.values.transpose(axes_numbers)
```

Comment:
shouldn’t u remove the previous line?

Reply:
Yah, thought I had removed this no-longer-needed bit, left a line behind.  Will fix.